spec/poparser/po_spec.rb

```diff
@@ -20,6 +20,13 @@
     expect(@po << entry).to be_a_kind_of PoParser::Po
   end
 
+  it 'should be able to remove an entry from Po' do
+    entries = [entry, entry.dup]
+    @po << entries
+    expect(@po.delete(@po.entries.first)).to be_a_kind_of PoParser::Po
```

Comment:
with changing the delete method to default ruby convention, this can be changed to
```suggestion
    expect(@po.delete(entry)).to eq entry
    expect(@po.delete(entry).to eq nil
```